messaging/src/main/java/org/axonframework/messaging/eventhandling/SimpleEventHandlingComponent.java

```diff
@@ -50,6 +53,8 @@ public class SimpleEventHandlingComponent implements EventHandlingComponent {
     private final ConcurrentHashMap<QualifiedName, List<EventHandler>> eventHandlers = new ConcurrentHashMap<>();
     private final SequencingPolicy sequencingPolicy;
 
+    private final List<ResetHandler> resetHandlers = new CopyOnWriteArrayList<>();
```

Comment:
If the majority feels strongly against having something like a `QualifiedName` for the `ResetHandlerRegistry#subscribe(ResetHandler)` method, we can always switch to a `Set` here.

Reply:
Awesome, thanks for that!